apps/documentation/src/app/pages/components/timeline-range-selector/demo/timeline-range-selector-demo.component.ts

```diff
@@ -1,13 +1,23 @@
+import { CommonModule } from '@angular/common';
 import { Component, OnInit } from '@angular/core';
-import { DataLayerManagerService } from '@elimuinformatics/ngx-charts-on-fhir';
+import {
+  DataLayerManagerService,
+  FhirChartConfigurationService,
+  FhirChartModule,
+  TimelineRangeSelectorModule,
+} from '@elimuinformatics/ngx-charts-on-fhir';
 
 @Component({
-  selector: 'example-timeline-range-selector-demo',
+  selector: 'app-timeline-range-selector',
   templateUrl: './timeline-range-selector-demo.component.html',
+  standalone: true,
+  imports: [CommonModule, FhirChartModule, TimelineRangeSelectorModule],
+  providers: [DataLayerManagerService, FhirChartConfigurationService],
 })
-export class TimelineRangeSelectorDemoComponent implements OnInit {
+export class TimelineRangeSelectorComponent implements OnInit {
```

Comment:
we should still include "Demo" in the component name (and "-demo" in the selector), to distinguish it from the actual charts-on-fhir component.